dali/sasha/packetstore.cpp

```diff
@@ -240,7 +240,7 @@ class CDataPacketTransaction: extends CInterface
     {
         CriticalBlock block(store.lockingsect);
         if (locked) {
-            LOG(MCerror, unknownJob, "Warning: transaction previously locked (lockRead)");
+            DBGLOG("Warning: transaction previously locked (lockRead)");
```

Comment:
these and others below that were LOG(MCerror, ..
look like they should be IERRLOG